logstash-core/logstash-core.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'puma', '~> 2.16', '>= 2.16.0'
   gem.add_runtime_dependency "jruby-openssl", "0.9.13" # Required to support TLSv1.2
   gem.add_runtime_dependency "chronic_duration", "0.10.6"
-  gem.add_runtime_dependency "jruby-monitoring", '~> 0.1'
+  gem.add_runtime_dependency "jruby-monitoring", '~> 0.3.1'
```

Comment:
I'm ok with this change, but I must warn that it changes the current behaviour:

```
jruby-1.7.23 :001 > Gem::Requirement.new("~> 0.3.1").satisfied_by?(Gem::Version.new("0.5"))
 => false 
jruby-1.7.23 :002 > Gem::Requirement.new("~> 0.1").satisfied_by?(Gem::Version.new("0.5"))
 => true 
```

An alternative would be to do:

```
gem.add_runtime_dependency "jruby-monitoring", '~> 0.1', '>= 0.3.1'
```